catalyst/callbacks/__init__.py

```diff
@@ -23,6 +23,7 @@
     ILRUpdater,
     LRFinder,
 )
+from catalyst.callbacks.aggregation import MetricAggregationCallback
```

Comment:
btw, add this callback to the docs please, here - https://github.com/catalyst-team/catalyst/blob/dev/docs/api/callbacks.rst